module-apps/application-messages/windows/CallSMSTemplatesWindow.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "CallSMSTemplatesWindow.hpp"
+
+namespace gui
+{
+
+    CallSMSTemplatesWindow::CallSMSTemplatesWindow(app::ApplicationCommon *app, const std::string &windowName)
+        : SMSTemplatesWindow(app, windowName)
+    {
+        buildInterface();
```

Comment:
AFAIU this function is called in the constructor of the base class so it's called twice. Or am I missing something?

Reply:
I just copied it from the parent window - either way I'll just use two times the same window with different names - this will do the same :)

Reply:
resolving as it's no more related - the file has been deleted